StringTools.csproj fails to build if 3.5 SP1 is not installed

Failing to build MSBuild on a machine with no 3.5 SP1 installed:

```
C:\msbuild2\.dotnet\sdk\5.0.102\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.FrameworkReferenceResolution.targets(3
59,5): error MSB3645: .NET Framework v3.5 Service Pack 1 was not found. In order to target ".NETFramework,Version=v3.5"
, .NET Framework v3.5 Service Pack 1 or later must be installed. [C:\msbuild2\src\StringTools\StringTools.csproj]
```

If I pass ` /p:BypassFrameworkInstallChecks=true` then it works fine:
![image](https://user-images.githubusercontent.com/679326/105803192-ccece900-5f51-11eb-88c8-4af92cf478cf.png)

--- a/src/StringTools/StringTools.csproj
+++ b/src/StringTools/StringTools.csproj
@@ -15,7 +15,12 @@
     <SemanticVersioningV1>true</SemanticVersioningV1>
 
     <AssemblyName>Microsoft.NET.StringTools</AssemblyName>
-    <AssemblyName Condition="'$(TargetFramework)' == 'net35'">Microsoft.NET.StringTools.net35</AssemblyName>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net35'">
+    <AssemblyName>Microsoft.NET.StringTools.net35</AssemblyName>
+    <!-- Disable Fx install checks as we're building against jnm2's 3.5 reference assemblies -->
+    <BypassFrameworkInstallChecks>true</BypassFrameworkInstallChecks>
   </PropertyGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' != 'net35'">

